boards/PDM/Src/App/App_SetPeriodicCanSignals.c

```diff
@@ -0,0 +1,17 @@
+#include "App_SetPeriodicCanSignals.h"
+#include "App_CanTx.h"
+#include "App_InRangeCheck.h"
+#include "App_ErrorCode.h"
+
+void App_SetPeriodicCanSignals_InRangeCheck(
```

Comment:
I started with one function for each inRangeCheck
```
App_SetPeriodicCanSignals_VbatInRangeCheck
App_SetPeriodicCanSignals_24vAuxInRangeCheck
...
```

But it seemed a bit redundant still so I just created one function for all the `struct InRangeCheck`